modules/cudf/src/series.ts

```diff
@@ -939,11 +938,11 @@ export class AbstractSeries<T extends DataType = any> {
   /**
    * @summary Return sub-selection from a Series using the specified integral indices.
    *
-   * @description Gathers the rows of the source columns according to `selection`, such that row "i"
-   * in the resulting Series's columns will contain row `selection[i]` from the source columns. The
-   * number of rows in the result series will be equal to the number of elements in selection. A
-   * negative value i in the selection is interpreted as i+n, where `n` is the number of rows in
-   * the source series.
+   * @description Gathers the rows of the source columns according to `selection`, such that row
+   * "i" in the resulting Series's columns will contain row `selection[i]` from the source
+   * columns. The number of rows in the result series will be equal to the number of elements in
+   * selection. A negative value i in the selection is interpreted as i+n, where `n` is the number
+   * of rows in the source series.
```

Comment:
I'm concerned about this. Obviously eslint thought to reformat. Is my estlint setting 1 character less than yours, or did it change but this file wasn't modified since it was changed?

Reply:
We use `clang-format-12` to format TS files. Do you have an eslint auto-formatter plugin installed too?

Reply:
I thought it was installed as part of node-rapids.workspace. Every time it saves it auto formats.

Reply:
As you can see [here](https://github.com/rapidsai/node/blob/main/node-rapids.code-workspace#L168), `clang-format` should be formatting `.ts` files. It's possible the ESLint plugin formatted the file if you saved it in an editor that wasn't opened via the VSCode workspace file?

Reply:
Reverted this.